boto3/docs/collection.py

```diff
@@ -181,7 +182,9 @@ def document_collection_method(section, resource_name, action_name,
             'method_description': (
                 'Creates an iterable of all %s resources '
                 'in the collection filtered by kwargs passed to '
-                'method.' % collection_model.resource.type),
+                'method.' % collection_model.resource.type +
+                'Collection will scope out to all resources '
```

Comment:
I think it's important to make the connection that if no filters are applied actions are applied to all resources by default perhaps something like:

```
A {type} collection will include all resources by default if no filters are provided, and extreme caution should be taken when performing actions on all resources.
```
I think we can use similar/the same wording for the above.